src/middleware/indexer.js

```diff
@@ -1,9 +1,11 @@
 const { Pool } = require('pg');
 const Cache = require('node-cache');
 const bunyan = require('bunyan');
+const fetch = require('node-fetch');
 
 const {
     BRIDGE_TOKEN_FACTORY_ACCOUNT_ID = 'factory.bridge.near',
+    GRAPHQL_URL = 'https://near-queryapi.api.pagoda.co/v1/graphql',
```

Comment:
Does this handle `testnet` queries as well?

Reply:
Ah that's a good point, it does not. We may need to parameterise this in ECS, and use ExplorerDB in testnet, QueryAPI doesn't support testnet yet.

Reply:
thanks for the heads up, I changed to have a fallback for non-mainnet environments or if no results from graphql

Reply:
Do you actually want a fallback for mainnet queries that return no results from graphql? If the DBs are actually going to be unavailable after April 30th then I imagine you'd want to know sooner than later that there's a disparity in what QueryAPI is returning. Also saves you the effort of needing to come back in < 2 weeks to remove the fallback 🙂 

Are the testnet indexer DBs also going to be unavailable after April 30th? It's sounding like there will be an indeterminate gap in coverage for testnet. I don't know how severe the real impact of this would be, it's probably safe to assume there isn't a lot of account recovery, staking, etc. coming from testnet environments. It would almost certainly break parity for wallets supporting testnet but it might not be a dealbreaker in the short term.

Reply:
Good questions, yes we want to keep the fallback, April 30th is the desired deadline, but there are other endpoints that needs to be migrated before we turn off the DB instance.

Testnet indexer DBs will stay, there is no deadline or workstream yet to deprecate them.